tests/test-cases/Group26-Custom-ISO/26-02-Centos6.9-Volume.robot

```diff
@@ -0,0 +1,254 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 26-02 - Centos6.9 Volume
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  bootstrap-iso=bin/bootstrap-centos-6.9.iso
```

Comment:
Please pass bootstrap-iso as a robotframework parameter, the test case should be able to test any custom bootstrap iso.